contrib/test.sh

```diff
@@ -1,10 +1,11 @@
-#!/bin/sh -ex
+#!/bin/bash -ex
 
-FEATURES="serde serde-std"
+# Combination of features to test, should be every features combination but std and core2 together
+# note std has a comma in the end so that following regex avoid matching serde-std
+FEATURES=("" "std," "use-core2" "use-core2-std" "std,use-core2" "std,serde-std" "use-core2,serde-std")
```

Comment:
serde added in 7b909c986f4490411c2773900108a91e115f922c, I forget about it because it's used but not listed in the Cargo.toml feature section...

Not sure if other combination are needed though